src/overlays/actors/ovl_Dm_Nb/z_dm_nb.c

```diff
@@ -9,7 +15,10 @@ void DmNb_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void DmNb_Update(Actor* thisx, GlobalContext* globalCtx);
 void DmNb_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+UNK_TYPE func_80C1DED0(DmNb* this, s32 arg1);
+void func_80C1DF18(DmNb* this, GlobalContext* globalCtx);
+void DmNb_UnkActorDraw(GlobalContext* globalCtx, s32 limbIndex, Actor* actor);
```

Comment:
Looks like those forward declarations are not needed.

Reply:
I'm personally OK with verbose forward declarations. They read as a list of functions available to the Actor system.